src/Features/Core/Portable/MetadataAsSource/MetadataAsSourceFileService.cs

```diff
@@ -268,7 +294,7 @@ private static void TryDeleteFolderWhichContainsReadOnlyFiles(string directoryPa
         {
             try
             {
-                foreach (var fileInfo in new DirectoryInfo(directoryPath).EnumerateFiles("*", SearchOption.AllDirectories))
+                foreach (var fileInfo in new DirectoryInfo(directoryPath).EnumerateFiles(" *", SearchOption.AllDirectories))
```

Comment:
Is this added space consequential?

Reply:
wat!

Reply:
great catch.  no clue how that happened :(